.github/workflows/python-integration-tests.yml

```diff
@@ -0,0 +1,54 @@
+#
+# This workflow will run all python integrations tests.
+#
+
+name: python-integration-tests
+
+on:
```

Comment:
nit: i think it'd be good to be consistent between the two. I think on push makes most sense, do we think the schedule will help identify issues?

Reply:
I could see it help identify breaking requirements changes from external packages. I'd prefer locking them down though for that reason :)

Reply:
Removing this file to PR separately. Will pick up this conversation there.